src/java/fr/paris/lutece/plugins/appointment/service/EntryService.java

```diff
@@ -575,6 +575,8 @@ public static void getHtmlEntry( Map<String, Object> model, int nIdEntry, String
             }
             template = AppTemplateService.getTemplate( entryTypeService.getTemplateHtmlForm( entry, bDisplayFront ), locale, model );
             stringBuffer.append( template.getHtml( ) );
+            
+            model.remove( MARK_STR_LIST_CHILDREN );
```

Comment:
rather than removing this mark, perhaps we should remove the code which adds it before?

Reply:
This method is called in a for loop where we use the same **model** variable.
So when this information is added, it will be used in building the template (two lines before).
If we don't remove this information juste after or at the beginning of the method, it will be present in the following iterations and used in cases where it should not.